app/controllers/openid_connect/logout_controller.rb

```diff
@@ -38,6 +38,19 @@ def delete
       end
     end
 
+    def create
```

Comment:
Also I'd expect to see a change in `routes.rb` for this too?

Reply:
I thought about doing that, but wasn't sure if what our conventions were around that. Will fix.

Reply:
After some digging, I am going to leave this as is, but I will extrapolate the logic in `index` and `create` to a separate private method. 

Rails explicitly doesn't support mapping routes to the same action nor cross site forgery protection for `GET` requests. As I am not confident how Rails interprets the `alias/alias_method` for action methods or how it may or may not interfere with the low-level monkey-patching Rails does under the hood at runtime, I worry that the way `alias_method` works will produce unpredictable results.

Reply:
> Rails explicitly doesn't support mapping routes to the same action
I think the `routes.rb` would allow that, we do that in a few places, here's one example

https://github.com/18F/identity-idp/blob/050cca7fdeaad1a9342ee6e5d530162b9d0ae306/config/routes.rb#L41-L42

Reply:
I don't feel super strongly as far as duplicating the get/post vs. the match, I think either is fine?

Reply:
Correction: I should've said that [Rails strongly recommends against mapping routes to the same action](https://guides.rubyonrails.org/routing.html#http-verb-constraints):

> Routing both GET and POST requests to a single action has security implications. In general, you should avoid routing all verbs to an action unless you have a good reason to.

I can try it out on my local and see if anything weird pops up, but in general, I prefer to keep to Rails conventions to reduce cognitive overhead.

Reply:
Seems fine to stick with convention then imo 👍🏼

Reply:
Fixed. This now uses one action and the `route.rb` rules have been updated to reflect this.